script/release.sh

```diff
@@ -134,6 +135,13 @@ git_pull() {
   cd ..
 }
 
+
+git_sync() {
+  cd $CLI
+  git pull upstream
```

Comment:
Hmm. It *should* work, I just tested it out and it pulled an updated branch of gh-pages I recently commited to. I'll update the script!